### ISSUE TITLE
Authentication with Web UI?

### ISSUE THREAD

**Issue by pixelskribe** [itsgoingd/clockwork#279]
Authentication is working with browser extensions, is the Web UI unsupported?  When accessing the web ui I am getting a "The page isn’t redirecting properly" error.

**Comment by pixelskribe** [comment]
I am not sure if it has something to do with my server's configuration, or if it is just a bug, but I discovered it only happens when the dark theme configuration setting is set to `true`.
```
'web_dark_theme' => env('CLOCKWORK_WEB_DARK_THEME', true),
```
When requesting `http://localhost/__clockwork/app?dark`  the response is a "302 Found" redirect which never resolves and gets repeated 21 times until it times out or something. 

If, however,  I leave the dark theme setting set to `false` and manually add the "dark" query parameter i.e. `app?dark` everything works just fine.  
```
![image](https://user-images.githubusercontent.com/8276518/44254229-e4fda180-a20a-11e8-8c04-6dd6c0024d30.png)

**Comment by itsgoingd** [comment]
Hey, can confirm this bug when used with Laravel versions before 5.5, will be fixed in 3.0.2 this week.
